File: sample-code/fibonacci.erl
%% Fibonacci (Erlang)

-module(fib).
-export([fib/1, main/0]).

fib(N) when N < 2 -> N;
fib(N) -> fib(N - 1) + fib(N - 2).

main() ->
  lists:foreach(fun(I) -> io:format("~p ~p~n", [I, fib(I)]) end, lists:seq(0, 11)).

File: sample-code/fibonacci.ex
# Fibonacci (Elixir)

defmodule Fib do
  def fib(n) when n < 2, do: n
  def fib(n), do: fib(n - 1) + fib(n - 2)
end

Enum.each(0..11, fn i ->
  IO.puts("#{i} #{Fib.fib(i)}")
end)
